tests/atest/rules/misc/empty-variable/test.robot

```diff
@@ -0,0 +1,9 @@
+*** Variables ***
```

Comment:
add also:
```
${MULTILINE_WITH_EMPTY}
...    ${EMPTY}
${MULTILINE_NO_VALUE}
...
${MULTILINE_WITH_EMPY_LINES}   Value
...  # I think it should be just empy line, without value - but I'm not sure
...    ${EMPTY}
```

Reply:
I tested it and it is... strange. You're right that empty string appeared in the second line of `${MULTILINE_WITH_EMPY_LINES}` but the same happened to `${MULTILINE_NO_VALUE}`. So my part of the code `if not node.value` will not catch it because actually this node has `value` field with 1 value which is `''` (empty string). 
I will adjust the code to catch those cases. Tell me what you think.